packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -133,22 +143,42 @@ export default function Sidebar({
           <Nav>
             <SidebarUserButton sidebarRendered={sidebarRendered} />
             {newSquadButtonVisible && (
-              <div className="flex">
-                <Button
-                  buttonSize={ButtonSize.Small}
-                  icon={<PlusIcon />}
-                  iconOnly={!sidebarExpanded}
-                  className={classNames(
-                    'mt-0 laptop:mt-2 mb-4 btn-primary-cabbage flex flex-1',
-                    sidebarExpanded ? 'mx-3' : 'mx-1.5',
-                  )}
-                  textPosition={
-                    sidebarExpanded ? 'justify-start' : 'justify-center'
+              <div className="flex mt-0 laptop:mt-2 mb-4">
+                <SimpleTooltip
+                  content={
+                    <>
+                      You can always
+                      <br /> open a squad here
+                    </>
                   }
-                  onClick={() => openSquadBetaModal({ origin: Origin.Sidebar })}
+                  placement="right"
+                  offset={newSquadTooltipOffset}
+                  sticky
+                  plugins={[sticky]}
+                  visible={newSquadTooltipTutorial.isActive}
```

Comment:
This could be used to render it as well right?

Reply:
@rebelchris you mean `newSquadTooltipTutorial.isActive && <SimpleTooltip />`?

Reply:
Yeah exactly, would that not be better then using a seperate state?

Reply:
It should actually be show for 4 seconds, so additional logic is required. Will update.